browser_history/browsers.py

```diff
@@ -113,22 +204,25 @@ class Chromium(Browser):
 
     history_file = Chrome.history_file
     bookmarks_file = Chrome.bookmarks_file
+    cookies_file = Chrome.cookies_file
 
     history_SQL = Chrome.history_SQL
+    cookies_SQL = Chrome.cookies_SQL
 
     bookmarks_parser = Chrome.bookmarks_parser
+    cookies_parser = Chrome.cookies_parser
 
 
 class Firefox(Browser):
     """Mozilla Firefox Browser
 
-    Supported platforms:
+        Supported platforms:
 
-    * Windows
-    * Linux
-    * Mac OS
+        * Windows
+        * Linux
+    -    * Mac OS
```

Comment:
What's the `-` for?

Reply:
I dont think I did that xD, Ill take it out, No clue how it got there :joy:

Reply:
lol